Creazione Verbale (#244)
CodingCowboys10/kmai: 3-PB/Documentazione_interna/Verbali/Verbale_2024_03_25_v1.0/resoconto.tex

\section{Resoconto} \label{sec:resoconto}
\subsection{Discussioni} \label{subsec:resdiscussione}
\begin{enumerate}

>     \item La riunione inizia con la fase di retrospettiva in cui viene evidenziato il significativo come la maggior parte degli obiettivi individuati all'inizio dello sprint precedente siano stati portati a termine. Si è posta particolare attenzione sul Manuale Utente e sulla Specifica Architetturale che necessita ancora di aggiornamenti. In questa fase si sono inoltre preventivate un numero maggiore di ore di verifica per far fronte ai numerosi interventi sui Documenti e alla fine della codifica dei Test.

Reviewer: ```suggestion
    \item La riunione inizia con la fase di retrospettiva, in cui viene evidenziato come la maggior parte degli obiettivi individuati all'inizio dello sprint precedente siano stati portati a termine. Si è posta particolare attenzione sul MVP, ora quasi terminato, e sui documenti Manuale\_utente e Specifica\_architetturale, che necessitano ancora di aggiornamenti. In questa fase si sono inoltre preventivate un numero maggiore di ore di verifica, per far fronte ai numerosi interventi sui documenti e alla fine della codifica dei test.
```

>     \item La discussione verte sullo stato finale del MVP evidenziando gli ottimi miglioramenti rispetto allo sprint precedente. Si è visto rapidamente il prodotto per verificare il corretto funzionamento. Dato lo stato terminale del MVP è stata richiesta una riunione al proponente fissata il giorno 26/03/2024 per l'accettazione del prodotto.

Reviewer: ```suggestion
    \item 
La discussione verte sullo stato quasi finale del MVP, evidenziando gli ottimi miglioramenti rispetto allo sprint precedente. Si è visionato e testato attentamente il prodotto, per verificare il suo corretto funzionamento. Dato lo stato terminale del MVP, è stata richiesta una riunione al proponente fissata al giorno 2024/03/26 per l'accettazione del prodotto.
```

>     \item Si è terminata la riunione discutendo riguardante l'avanzamento e la codifica dei test. Viene evidenziato come la parte di back-end disponga di un elevato numero di test mentre è ancora in fase di sviluppo la corrispettiva parte per il front-end. Si è posta una nota problematica per quanto riguarda la codifica di quest'ultimi.

Reviewer: ```suggestion
    \item Si è terminata la riunione discutendo sull'avanzamento nella codifica dei test. Viene evidenziato come la parte di back-end disponga di un elevato numero di test, mentre è ancora in fase di sviluppo la corrispettiva parte per il front-end. Si è posta una nota problematica per quanto riguarda la codifica di quest'ultimi.
```

\end{enumerate}

\subsection{Azioni da intraprendere}
{
    \setlength{\tabcolsep}{10pt}
            \renewcommand{\arraystretch}{1.5}
            \rowcolors{2}{oddrow}{evenrow}
            \begin{tabularx}{\textwidth}{| l | l | l | X |}
                 \hline
                 \rowcolor{headerrow}\textbf{\textcolor{white}{Codice issue}} & \textbf{\textcolor{white}{Assegnatario}} & \textbf{\textcolor{white}{Scadenza}} & \textbf{\textcolor{white}{Descrizione}} \\
                 \hline

>                  cc-259 & Marco Dolzan & 2024/04/01 & Stesura DataSources. \\

Reviewer: ```suggestion
                 cc-259 & Anna Nordio & 2024/04/01 & Stesura DataSources. \\
```

                 \hline

>                  cc-269 & Giovanni Menon & 2024/04/01 & Stesura Verbale 25/03/2024. \\

Reviewer: ```suggestion
                 cc-269 & Giovanni Menon & 2024/04/01 & Stesura Verbale 2024/03/25. \\
```
